bakerydemo/jinja2/base.html

```diff
@@ -1,43 +1,37 @@
-{% load navigation_tags static wagtailuserbar %}
 <!DOCTYPE html>
 <html lang="en">
     <head>
         <meta charset="utf-8">
         <title>
             {% block title %}
-                {% if page.seo_title %}
-                    {{ page.seo_title }}
-                {% else %}
-                    {{ page.title }}
-                {% endif %}
+                {{ page.seo_title if page.seo_title else page.title }}
             {% endblock %}
             {% block title_suffix %}
-                | {{ settings.base.SiteSettings.title_suffix }}
+                | {{ settings("base.SiteSettings").title_suffix }}
             {% endblock %}
         </title>
-        <meta name="description" content="{% if page.search_description %}{{ page.search_description }}{% endif %}">
+        <meta name="description" content="{{ page.search_description if page and page.search_description }}">
```

Comment:
This isn’t good in the base bakerydemo as views that aren’t page will have the meta tag with an empty value.